lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -619,19 +614,17 @@ void TypeAnalysis::visitOperation(Operation *op,
           AtenBitwiseNotOp, AtenToPrimDeviceOp, AtenCpuOp, AtenContiguousOp,
           AtenDetachOp, AtenMaskedFill_ScalarOp, AtenCopyOp, AtenCumsumOp,
           AtenLayerNormOp, AtenClampOp, AtenClampMinOp, AtenClampMaxOp,
-          AtenNegOp, AtenFloorOp, Aten_SoftmaxBackwardDataOp, AtenDropoutOp,
-          AtenTanhBackwardOp, AtenHardtanhBackwardOp,
```

Comment:
There is already a patch that will handle the ops here: https://github.com/llvm/torch-mlir/pull/1895. Can you undo these changes to avoid conflicts?